packages/shared/src/cli/base.ts

```diff
@@ -28,6 +28,15 @@ export abstract class BaseCommandLine extends CommandLineParser {
     description: 'Show extra extra logging detail',
   });
 
+  toolName: string;
+  toolDescription: string;
```

Comment:
Are we using this `toolDescription` anywhere?

Reply:
Nope figured for completeness we should have it though